internal/cobra-cmds/version.go

```diff
@@ -22,7 +22,7 @@
  * SOFTWARE.
  */
 
-package kache
+package cobracmds
```

Comment:
this should be reverted, we have separate versions for server and cli. Also fix consecutive files.